Package.swift

```diff
@@ -149,8 +149,8 @@ let package = Package(
       "10.0.0" ..< "11.0.0"
     ),
     .package(
-      url: "https://github.com/google/GoogleUtilities.git",
```

Comment:
I'll update this to use the proper GoogleUtilities tag once it's merged, tagged, and released.

Reply:
Hi @harlanhaskins, GoogleUtilities 8.1.0 has now been released